docs/docs-beta/docs/dagster-plus/deployment/code-locations.md

```diff
@@ -3,4 +3,237 @@ title: "Code locations"
+
+- [GitHub repository with Dagster+ Serverless](https://github.com/dagster-io/dagster-cloud-serverless-quickstart/)
+- [GitHub repository with Dagster+ Hybrid](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/)
+- [Gitlab CICD for Dagster+ Serverless](https://github.com/dagster-io/dagster-cloud-action/blob/main/gitlab/serverless-ci.yml)
+- [Gitlab CICD for Dagster+ Hybrid](https://github.com/dagster-io/dagster-cloud-action/blob/main/gitlab/hybrid-ci.yml)
+
+
+Overall, the Git repository should contain:
+
+1. Your Dagster code, structured as a Python module. The repository might look like this:
+
+```bash
+README.md
+dagster_cloud.yaml
+Dockerfile
+/.github
+    /workflows
+        dagster-cloud-deploy.yml
+setup.py
+pyproject.toml
+/quickstart
+    __init__.py
+    definitions.py
+    /assets
+        ...
+    /resources
+        ...
+```
+
+2. A `dagster_cloud.yaml` file with the settings for your code location. Here is an example:
+
+```yaml
+locations:
+  - location_name: quickstart
+    code_source:
+      package_name: quickstart
+```
+
+3. A CICD workflow file that contains the steps for adding your code location. Here is an example workflow file for a Dagster+ Hybrid organization:
+
+```yaml
+variables:
+  DAGSTER_CLOUD_ORGANIZATION: <organinization-name>
+  DAGSTER_PROJECT_DIR: .
+  IMAGE_REGISTRY: <account-id>.dkr.ecr.us-west-2.amazonaws.com/<image-name>
+  IMAGE_TAG: $CI_COMMIT_SHORT_SHA-$CI_PIPELINE_ID
+
+stages:
+  - build
+  - deploy
+
+build:
+  stage: build
+  image: docker:latest
+  services:
+    - docker:dind
+  before_script:
+    # # For Gitlab Container Registry
+    # - echo $CI_JOB_TOKEN | docker login --username $CI_REGISTRY_USER --password-stdin $REGISTRY_URL
+    # # For DockerHub
+    # - echo $DOCKERHUB_TOKEN | docker login --username $DOCKERHUB_USERNAME --password-stdin $REGISTRY_URL
+    # # For AWS Elastic Container Registry (ECR)
+    # - apk add --no-cache curl jq python3 py3-pip
+    # - pip install awscli
+    # - echo $AWS_ECR_PASSWORD | docker login --username AWS --password-stdin $IMAGE_REGISTRY
+    # # For Google Container Registry (GCR)
+    # - echo $GCR_JSON_KEY | docker login --username _json_key --password-stdin $REGISTRY_URL
+  script:
+    - docker build . -t $IMAGE_REGISTRY:$IMAGE_TAG
+    - docker push $IMAGE_REGISTRY:$IMAGE_TAG
+
+deploy:
+  stage: deploy
+  dependencies:
+    - build
+  image: ghcr.io/dagster-io/dagster-cloud-action:0.1.43
+  script:
+    - dagster-cloud deployment add-location --deployment prod --image
+    $IMAGE_REGISTRY:$IMAGE_TAG --location-name quickstart --package-name quickstart
+```
+
+## Adding a new code location to a Git monorepo
+
+Many organizations use a Git monorepo to contain multiple Dagster projects. Here is an example of DagsterLab's own [internal data engineering Git repository](https://github.com/dagster-io/dagster-open-platform). To add a new location to a monorepo, you will create a new directory containing:
```

Comment:
```suggestion
Many organizations use a Git monorepo to contain multiple Dagster projects. Here is an example of DagsterLab's own [internal data engineering Git repository](https://github.com/dagster-io/dagster-open-platform).

To add a new code location to a monorepo, create a new directory that contains:
```

- Suggesting we add `code` to `location` to make it specific and consistent
- Also that we remove `you will` and instead just use understood you to make this a command
- Adding a line break to visually break up the copy